source/chapters/user_interface.rst

```diff
@@ -10,18 +10,18 @@ You are going to want to get very familiar with this interface because it is
 the main way to use Mixxx. In this chapter, we present the default interface of
 Mixxx and describe its elements, knobs and faders.
 
-.. figure:: ../_static/Mixxx-200-Latenight.png
+.. figure:: ../_static/Mixxx-200-Deere.png
```

Comment:
let's use lowercase file names. Also, the filename indicates the screenshot is from mixxx 2.0.0, but it's not. I'd prefer a naming scheme like this:

```suggestion
.. figure:: ../_static/2.2/interface.png
```

Reply:
True. I agree. The names were a bit too long.